tests/e2e/apps/hop/app.go

```diff
@@ -34,13 +34,39 @@ import (
 	"google.golang.org/grpc"
 
 	"istio.io/istio/tests/e2e/apps/hop/config"
+	"istio.io/istio/tests/e2e/framework"
+	"istio.io/istio/tests/e2e/util"
 )
 
 var (
-	timeout = flag.Duration("timeout", 15*time.Second, "Request timeout")
-	version = flag.String("version", "", "Server version")
+	timeout     = flag.Duration("timeout", 15*time.Second, "Request timeout")
+	version     = flag.String("version", "", "Server version")
+	hopYamlTmpl = "tests/e2e/framework/testdata/hop.yam.tmpl"
```

Comment:
Since this seems like a configurable variable perhaps we can also make it configurable e.g
```go
hopYamlTmplPath = flag.String('hop-yaml-tmpl", "tests/e2e/framework/testdata/hop.yam.tmpl", "Path to the hop yaml template")
```
However, I don't think this matters since the original used a const for hopYamlTmpl.